_playwright-tests/UI/SnapshotRepo.spec.ts

```diff
@@ -102,4 +103,50 @@ test.describe('Snapshot Repositories', () => {
       await expect(row).not.toBeVisible();
     });
   });
+
+  test('test snapshot manual trigger', async ({ page }) => {
+    await navigateToRepositories(page);
+    await closePopupsIfExist(page);
+    const repoNamePrefix = 'snapshot-manual-trigger';
+    const randomName = () => `${(Math.random() + 1).toString(36).substring(2, 6)}`;
+    const repoName = `${repoNamePrefix}-${randomName()}`;
+    const url = randomUrl();
+    await deleteAllRepos(page, `&search=${repoNamePrefix}`);
+
+    await test.step('Create a repository', async () => {
+      await page.getByRole('button', { name: 'Add repositories' }).first().click();
+      await expect(page.getByRole('dialog', { name: 'Add custom repositories' })).toBeVisible();
+      await page.getByLabel('Name').fill(`${repoName}`);
+      await page.getByLabel('Introspect only').click();
+      await page.getByLabel('URL').fill(url);
+      await page.getByRole('button', { name: 'Save', exact: true }).click();
+      // Need to wait for introspection task to finish
+      await page.waitForTimeout(7000);
```

Comment:
the task is already finished when the status becomes valid (and the test waits for that 2 lines below), so this can be removed 🗑️

Reply:
yes I agree !